packages/cms-lib/lib/config/index.js

```diff
@@ -0,0 +1,36 @@
+const {
+  createEmptyConfigFile,
+  deleteEmptyConfigFile,
+  getAndLoadConfigIfNeeded,
+  getConfig,
+  getConfigPath,
+  isTrackingAllowed,
```

Comment:
See `isTrackingAllowed` in particular feels out of place when it comes to the `config/file`.

Reply:
I agree. I will move it somewhere more appropriate.